packaging/shellai.spec

```diff
@@ -0,0 +1,42 @@
+Name:           shellai
+Version:        0.1.0
+Release:        1%{?dist}
+Summary:        A simple wrapper to interact with RAG
+
+License:        Apache-2.0
+URL:            https://github.com/rhel-lightspeed/shellai
+Source0:        %{url}/archive/v%{version}/%{name}-%{version}.tar.gz
+# noarch because there is no extension module for this package.
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+# Not need after RHEL 10 as it is native in Python 3.11+
+%if 0%{?rhel} && 0%{?rhel} < 10
+BuildRequires:  python3-tomli
+%endif
+
+Requires:       python3-requests
+Requires:       python3-pyyaml
+
+%description
+A simple wrapper to interact with RAG
+
+%prep
+%autosetup -n %{name}-%{version}
+
+%build
+%py3_build
```

Comment:
Might want to use the more modern pyproject_rpm_macros here. They should be in RHEL 9.

Reply:
This is only available for RHEL 10+ :( at least, some of the nice features. 

To avoid having too much conditionals for now inside this specfile, let's go simple